Ktisis/Interface/Editor/Properties/ActorPropertyList.cs

```diff
@@ -71,7 +89,111 @@ private void DrawActorTab(ActorEntity actor) {
 	
 	// Gaze tab
 
-	private void DrawGazeTab() {
-		
+	private unsafe void DrawGazeTab(ActorEntity actor) {
+		if (GazeTables == null)
+			GazeTables = new();
+
+		// work from existing gaze on ActorEntity or make a new one if its our first touch w them
+		var gaze = actor.Gaze != null ? (ActorGaze)actor.Gaze : new ActorGaze();
+		// if human actor, enable link/unlinked controls
+		bool isHuman = actor.GetHuman() != null;
+
+		var spacing = ImGui.GetStyle().ItemInnerSpacing.X;
+		var result = false;
+
+		using (var _disable = ImRaii.Disabled(this._ctx.Posing.IsEnabled)) {
```

Comment:
if you're not using the `_disable` property, you can simply do a discard assignment instead
```suggestion
		using (var _ = ImRaii.Disabled(this._ctx.Posing.IsEnabled)) {
```

but given this is a traditional using statement, no assignment is needed.
```suggestion
		using (ImRaii.Disabled(this._ctx.Posing.IsEnabled)) {
```